src/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './stringify';
+export * from './parse';
+
+// parser is a heavier dep, so it is not included in default exports. must import directly
+// export * from './parseWithPointers';
```

Comment:
We should probably just export the parse and parseWithPointers functions as well for this library. It's the primary purpose. And take the corresponding sentence out of the readme about it not being exported from root.